haskell/stack.yaml

```diff
@@ -30,13 +30,6 @@ extra-deps:
 - show-type-0.1.1
 - git: https://github.com/int-index/caps.git
   commit: c5d61837eb358989b581ed82b1e79158c4823b1b
-# tezos-bake-monitor-lib and its deps which are used in morley-client
```

Comment:
It seems there are some dependencies of `tezos-bake-monitor-lib` below that are not needed anymore. Could you please check whether there are useless extra-deps and delete them?

Reply:
Done